src/databasetasks.h

```diff
@@ -47,6 +47,7 @@ class DatabaseTasks : public ThreadHolder<DatabaseTasks>
 		void threadMain();
 	private:
 		void runTask(const DatabaseTask& task);
+		void flushInternal();
```

Comment:
@djarek check my last commit, i have also test the call to g_databaseTasks.flush() and g_databaseTasks.shutdown()